sic_pg_extract.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Sep 11 13:50:53 2024
+This script extracts numeric pg codes that map to more than one pg_alpha, 
+depending on the sic code
+@author: zoring
```

Comment:
It's usful to have a docstring at the top of the file, though we haven't been putting the author and date on our work (you can of course determine this from git). Up to you if you want to keep this more formal/ usual style.